### ISSUE TITLE
Incorrect concatination of asset file names and directory path

### ISSUE THREAD

**Issue by MeiKatz** [ztoben/assets-webpack-plugin#88]
# Issue description
Hi, I tried this plugin and my `publicPath` and my chunk names are concatenated without a slash in between:
```javascript
publicPath: "/assets",
assetsByChunkName: {
  main: [
    'app-c72b8e3607a4176d7a01.js',
    'app-c72b8e3607a4176d7a01.js.map'
  ],
  vendor: [
    'vendor-ffe9523a5337969b51e2.js',
    'vendor-ffe9523a5337969b51e2.js.map'
  ]
}
```
And the resulting JSON file looks like this:
```json
{"main":{"js":"/assetsapp-c72b8e3607a4176d7a01.js"},"vendor":{"js":"/assetsvendor-ffe9523a5337969b51e2.js"}}
```
In think the problem is this line: https://github.com/kossnocorp/assets-webpack-plugin/blob/master/index.js#L64
```javascript
typeMap[typeName] = assetPath + asset
```

# Solution
Either use the `path` package:
```javascript
typeMap[typeName] = path.join(assetPath, asset);
```
or only a slash between:
```javascript
typeMap[typeName] = assetPath + "/" + asset;
```
I think the first choice is the cleaner solution.

**Comment by AjayPoshak** [comment]
+1

**Comment by ztoben** [comment]
This should be fixed in the next release.

Edit: `3.6.2` is available on npm now.

**Comment by Pranab16** [comment]
@ztoben Your last fix broke my app in production. I was setting CDN url in assetPath something like https://cdn.com/ and asset being fileName. But this change results in https:/cdn.com//fileName. Any reason for this change? 
To solve #88, we can check if last char in assetPath is slash then just join assetPath and asset else ass slash between them.

**Comment by ztoben** [comment]
@Pranab16, fixed in `3.7.1`.
